src/main/java/lotto/controller/LottoController.java

```diff
@@ -1,41 +1,66 @@
 package lotto.controller;
 
-import lotto.domain.lotto.LottoGenerator;
-import lotto.domain.lotto.LottoGroup;
-import lotto.domain.lotto.WinningNumber;
-import lotto.domain.lotto.AnalysedLottos;
+import lotto.domain.lotto.*;
+import lotto.domain.number.ManualPurchasesNumber;
+import lotto.domain.number.Number;
 import lotto.domain.number.PayOut;
 import lotto.view.InputView;
 import lotto.view.OutputView;
 
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+
 public class LottoController {
     private static final LottoGenerator lottoGenerator = new LottoGenerator();
 
     public static void run() {
         PayOut payOut = getPayOutFromUser();
 
-        LottoGroup lottoGroup = createLottosAccordingToTheAmount(payOut);
+        ManualPurchasesNumber manualLottoCount = getManualLottoCountFromUser(payOut);
+        List<String> manualLottos = getManualLottosFromUser(payOut, manualLottoCount);
 
-        WinningNumber winningNumber = new WinningNumber(
-                getLastWeekLottoNumberFromUser(),
-                getBonusNumberFromUser()
+        LottoGroup lottoGroup = createLottosAccordingToTheAmount(manualLottos,
+                payOut.subtractionUsingGameCount(manualLottoCount.getValueAsInt())
         );
 
-        lottoResultAnalysisAndPrint(winningNumber, lottoGroup, payOut);
+        createWinningNumberAndlottoResultAnalysisAndPrint( lottoGroup, payOut);
     }
 
     private static PayOut getPayOutFromUser() {
         OutputView.payout();
 
         PayOut payOut = new PayOut(InputView.getStringInputFromUser());
```

Comment:
oneline으로 리턴가능할거 같아요 🙏
```suggestion
        return new PayOut(InputView.getStringInputFromUser());
```